### PR TITLE
JTD-002 Create Project Table

### PR DESCRIPTION
**Pull request by james-liam-butler**

### CODE REVIEW

**Review by lukemKainos — APPROVED** [review]

File: src/main/resources/db/migration/V2__create_project_table.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+CREATE TABLE project (
+    `name` varchar(50),
```
Comment (lukemKainos) [inline src/main/resources/db/migration/V2__create_project_table.sql:3]: Is this your primary key?

Comment (james-liam-butler) [reply src/main/resources/db/migration/V2__create_project_table.sql:3]: no, new primary key is projectId

File: src/main/resources/db/migration/V2__create_project_table.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+CREATE TABLE project (
+    projectId int NOT NULL,
```
Comment (lukemKainos) [inline src/main/resources/db/migration/V2__create_project_table.sql]: Are you going to add the ids manually?

**Review by lukemKainos — CHANGES_REQUESTED** [review]

**Review by dana-kainos — CHANGES_REQUESTED** [review]
salesEmployeeID should be camel case

**Review by lukemKainos — CHANGES_REQUESTED** [review]
Fix this pr title

**Review by lukemKainos — APPROVED** [review]

**Review by dana-kainos — APPROVED** [review]

**Pull request merged**